packages/docs/content/4_utilities/float.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Float
+---
+
+import ResponsiveExample from '../../src/components/ResponsiveExample';
+
+Set an element's `float` property using the float utility.
+
+You can use the clearfix utility class to automatically clear your floats.
+
+**Format**: `.ds-u-float--[left|right|none]`
+
+**Responsive format**: `.ds-u-[breakpoint]-float--[left|right|none]`
+
+<EmbeddedExample>
+  <section class="example--wrapper">
+    <div class="ds-u-clearfix ds-u-padding--1 ds-u-border--1 ds-u-margin-bottom--2">
+      <p class="ds-u-float--left utility-example ds-u-padding--1">
+        <code>.ds-u-float--left</code>
+      </p>
+    </div>
+    <div class="ds-u-clearfix ds-u-padding--1 ds-u-border--1 ds-u-margin-bottom--2">
+      <p class="ds-u-float--right utility-example ds-u-padding--1">
+        <code>.ds-u-float--right</code>
+      </p>
+    </div>
+    <div class="ds-u-clearfix ds-u-padding--1 ds-u-border--1 ds-u-margin-bottom--2">
+      <p class="ds-u-float--none utility-example ds-u-padding--1">
+        <code>.ds-u-float--none</code>
+      </p>
+    </div>
+  </section>
+</EmbeddedExample>
```

Comment:
It's so nice to not have to wait for these embedded examples to load separately and then reflow the document!